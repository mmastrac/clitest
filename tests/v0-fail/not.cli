#!/usr/bin/env clitest --v0

$ printf "goodbye\nworld\n"
not {
    ! goodbye
}
! hello
! world
