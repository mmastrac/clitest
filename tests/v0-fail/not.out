Running <root>/not.cli ...

printf "goodbye\nworld\n"
---
goodbye
world
---
ERROR: pattern not at line <root>/not.cli:4 does not match output line "goodbye"
matching Sequence([Not(Literal(goodbye)), Literal(hello), Literal(world), End])
  matching Not(Literal(goodbye))
    matching Literal(goodbye)
    literal match: "goodbye" == "goodbye"

‚ùå FAIL

<root>/not.cli FAILED
Error: pattern not at line <root>/not.cli:4 does not match output line "goodbye"

